### PR TITLE
Introduce CI tests

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
- introduce unit tests & eslint check
- run tests & deploy automatically with Travis CI
- update README file
- add `--force` option

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+[*]
+end_of_line = lf
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
+
+[Makefile]
```
Comment (mrtnzlml) [inline .editorconfig]: This is not needed since we are not using makefiles.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+language: node_js
```
Comment (mrtnzlml) [inline .travis.yml:1]: Can you please add Travis-CI badge to the README file?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+node_js:
+- '8'
+- '6'
+- '4'
```
Comment (mrtnzlml) [inline .travis.yml]: No one cares about Node.js 4. Please remove it.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+script:
+- npm test
```
Comment (mrtnzlml) [inline .travis.yml]: We are using `yarn` so we should use it here as well. Travis knows yarn...

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+deploy:
+  provider: npm
+  email: jaroslav.kubicek@gmail.com
+  api_key:
+    secure: c5qZod9MXoBTGMe2sztNpvXjzgamgJi4Vm7BJSNzUHcWJkG87DPgtVD+pCc0eu2xORaoBYkriEH73VzfAMl+0Tn12EH2sWTV8D3D2xNOWA8hsWIOyUQi1ozo8cEXf5Y+9MbMDyuE8h4o9ltUj9KYHTvElpdRNlK+Uvz4NCZjOny6EQJTYl+cbO3/GRsgBvdcz5s8PK3CckyPc65SbnB48qYrA8PIhByQQH789hhFrV80NhXZnVaRvaVT5XL1vsJZH1OZxlMQVWnkI5HsVOAjMtDqMXvIWP+/QBy9O7Fue1MKFrzBT0iLwVqIoq8+uWUHImFpd41lsUB0z9tVP5FarfDJIw08HeM3pHwaZ1ZbE0qkO7Ew8MahSp2ND7mQnG61k49+askyeGuFKXsfoQ7vZ8a91IjgfI4tP2L0Hc84gvZZwY/cu28BcaBiGWBccfTcUkZoZZ5KjOpxpbrg1nyH/aOGtYN/J9/6MIs/FQt67wHo6UE/2MdNmLrLathse617pQOQQOpgsrmp6YfwHYaApKgxKcC7HU+1BA0iBkGR8RdmJ24SCqqEUtr32Af9AArsXTFoakOwm6dASR0vda5T1UWsRRq/WW+WLollRKZxFrnxSXxsL88sTS4DZv8s50EzWASBKrvHfKsTzqap4GdbcPwvSGQ0STY+HeYhmQy9h+E=
```
Comment (michalsanger) [inline .travis.yml:11]: is it safe to share this value?

File: README.md [hunk 6]
```diff
@@ -1,3 +1,29 @@
 # vault2env-js
 
-`vault2env --token=[vault-token] --addr=[vault-addr] --path=secret/path/to/envs`
\ No newline at end of file
+`vault2env` let's you easily load env variables from your Vault instance and
+save them in `.env` file
```
Comment (mrtnzlml) [inline README.md]: Please try to avoid line-breaks in the middle of the sentence. Use soft-wrap in your IDE instead...

**Review by mrtnzlml — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 5 above)
Comment (mrtnzlml) [reply .travis.yml:11]: Yes.

File: .travis.yml (hunk 2 above)
Comment (jaroslav-kubicek) [reply .travis.yml:1]: added

**Review by mrtnzlml — APPROVED** [review]

**Pull request merged**